doc/Support/Performance.md

```diff
@@ -174,3 +174,36 @@ For Nginx (1.9.5 and above) change `listen 443 ssl;` to `listen 443
 ssl http2;` in the Virtualhost config.
 
 For Apache (2.4.17 an above) set `Protocols h2 http/1.1` in the Virtualhost config.
+
+## PHP-opcache
+
+A lot of performance can be gained from setting up `php-opcache`correctly. 
+
+### Example CentOS 7 configuration for distributed pollers (PHP 7.2)
+
+Create a cache directory that is writable by the librenms user first:
+`sudo mkdir -p /tmp/cache && sudo chmod 775 /tmp/cache && sudo chown -R librenms /tmp/cache`
+
+In `/etc/php.d/opcache.ini` you can set the following: (Note, your opcache.ini location
+may differ a bit)
+
+```
+zend_extension=opcache
+opcache.enable=1
+opcache.enable_cli=1
+opcache.file_cache="/tmp/cache/"
+opcache.file_cache_only=0
+opcache.file_cache_consistency_checks=1
+opcache.memory_consumption=256
+```
+
+When setting this on the pollers there is no need to restart anything. 
+Just note that by using file-based caching as is the functionality explained here, you might
+in extreme cases face some form of caching issue. Although LibreNMS does not use any
+long-running php services which would make above-mentioned scenario unlikely, it is not impossible.
+On pollers, you could reboot your machine, or a little less extreme just `rm -rf /tmp/cache` and issue
+restart the librenms service. This should again create a new directory into which OPcache will place
+cache files.
+
+**Note: The file_cache method is not advised for the web servers. If you want to enable opcache on 
```

Comment:
This should be more prominent and note that some OS have separate php configs for cli and web.